Makefile

```diff
@@ -56,6 +56,16 @@ testdb: build
 	.${BINARY} -name "KVM Linux containerd" -repo "github.com/kubernetes/minikube/" -pr "16569" -in "testdata/testdb/KVM_Linux_containerd.json" -out_html "./out/kvm_linux_containerd_output.html" -details "0168d63fc8c165681b1cad1801eadd6bbe2c8a5c"
 	.${BINARY} -name "QEMU MacOS" -repo "github.com/kubernetes/minikube/" -pr "16569" -in "testdata/testdb/QEMU_macOS.json" -out_html "./out/qemu_macos_output.html" -details "0168d63fc8c165681b1cad1801eadd6bbe2c8a5c"
 
+.PHONY: testpgdb
+testpgdb: export DB_BACKEND=postgres
+testpgdb: export DB_PATH='host=k8s-minikube:us-west1:flake-rate user=postgres dbname=flakedbdev password=${DB_PASS}'
+testpgdb: build
+	rm -f ./out/output.html
+	rm -f ./out/output2.html 
```

Comment:
Should there also be a `rm -f ./out/output2NoDBPath.html`?

Reply:
It doesn't actually seem we need either. When I run it on my crd, the .json's and .html's have it so that they seem to be getting updated (their modified date updates when the code runs) which is different than the binary which stays the same. I had just kept those in as the original test target does but I don't think they are necessary (though for the .db's this would make sense to do as there is the actual testing of creating a database)